influxdb3_write/src/write_buffer/loader.rs

```diff
@@ -335,7 +336,7 @@ mod tests {
             PersistedSegment {
                 segment_id,
                 segment_wal_size_bytes: 201,
-                segment_parquet_size_bytes: 3398,
+                segment_parquet_size_bytes: 3458,
                 segment_row_count: 3,
                 segment_min_time: 10,
                 segment_max_time: 20,
```

Comment:
@pauldix - for some reason, updating to core changed the byte sizes of these parquet files. I don't know why, but I was assuming you based these numbers on what was outputted at the time that you wrote the test. Is the test maybe just a bit brittle? or were these numbers intentional?

Reply:
I updated to reflect the new values, and to get tests ✅, but just want to make sure I'm not cheating by doing so.